### DISCUSSION TITLE
Where I can find Filter instance to do multiple filter?

### DISCUSSION THREAD

**Discussion by val089** [invertase/react-native-firebase#7132] [Q&A]
https://rnfirebase.io/firestore/usage

 I want to create something like this:
 
 const snapshot = await firestore()
  .collection('Users')
  .where(Filter.and(Filter('user', '==', 'Tim'), Filter('email', '==', 'tim@example.com')))
  .get();

**Comment by coycoylaniba** [comment]
i cant find it also
